test/jest.setup.js

```diff
@@ -23,6 +23,13 @@ jest.mock('fs', () => require('jest-plugin-fs/mock'))
 // ensure a mocked openwhisk module for unit-tests
 jest.mock('openwhisk')
 
+// clear env variables
+delete process.env['WHISK_AUTH']
```

Comment:
Potential issue here, we might wipe out a user's existing config that they might rely on.
Perhaps we should restore after the tests?

Reply:
it just deletes it in process.env not in the environment

Reply:
it just deletes it in process.env not in the environment